scripts/rpm-ostree-completion.sh

```diff
@@ -0,0 +1,46 @@
+#/usr/bin/env bash
+
+# Copyright (C) 2018 Chris Weeks <chrisweeks@catalyst.net.nz>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Purpose: a simple bash completion script for rpm-ostree.
+
+_get_completion()
+{
+  local cur prev opts
+  COMPREPLY=()
+  cur="${COMP_WORDS[COMP_CWORD]}"
+  prev="${COMP_WORDS[COMP_CWORD-1]}"
+  nexttoprev="${COMP_WORDS[COMP_CWORD-2]}"
+  case ${prev} in
+    rpm-ostree)
+      if [ ${COMP_CWORD} -eq 1 ]; 
+      then 
+          COMPREPLY=( $(compgen -W "$(rpm-ostree --help | grep -e '^  ' | grep -v 'rpm-ostree' | sed 's/-h, //' | awk '{print $1}')" -- "${cur}") ); fi 
```

Comment:
Thanks for the feedback. I have altered the completion script to export variables for each completion type and to reference that variable if present, rather than invoking rpm-ostree on every execution. 

Example of variables created after using the completion script:

```shell
# declare | grep ^_rpmostree_command
_rpmostree_command_completions=$'compose\ncleanup\ndb\ndeploy\nrebase\nrollback\nstatus\nupgrade\nreload\nusroverlay\ncancel\ninitramfs\ninstall\nuninstall\noverride\nreset\nrefresh-md\nkargs\n--help\n--version'
_rpmostree_command_compose_completions=$'tree\ninstall\npostprocess\ncommit\n--help'
_rpmostree_command_compose_tree_completions=$'--help\n--repo=REPO\n--force-nocache\n--cache-only\n--cachedir=CACHEDIR\n--download-only\n--unified-core\n--proxy=PROXY\n--dry-run\n--output-repodata-dir=DIR\n--print-only\n--touch-if-changed=FILE\n--workdir=WORKDIR\n--add-metadata-string=KEY=VALUE\n--add-metadata-from-json=JSON\n--write-commitid-to=FILE\n--write-composejson-to=FILE'
_rpmostree_command_rollback_completions=$'--help\n--reboot\n--sysroot=SYSROOT\n--peer'
```

Test environment build:
```shell
docker run -it --rm fedora:latest
dnf -y update
dnf -y install rpm-ostree
curl --silent --output /etc/bash_completion.d/rpm-ostree-completion https://raw.githubusercontent.com/chrisweeksnz/rpm-ostree/patch-1/scripts/rpm-ostree-completion.sh
source /etc/bash_completion.d/rpm-ostree-completion
```